mix.exs

```diff
@@ -47,7 +47,8 @@ defmodule Avalanche.MixProject do
       {:nimble_options, "~> 0.4.0"},
       {:mentat, "~> 0.7.1"},
       {:plug, "~> 1.13"},
-      {:req, "~> 0.3.0"},
+      # {:req, "~> 0.3.0"},
```

Comment:
I think you can do `{:req, "~> 0.3.1"}` too?

Reply:
That will automatically update it to `0.3.2`, but apparently this works: `{:req, "0.3.1"}`. That'll force the specific version.

I'll make that update 👍🏼